docs/book/src/component-config-tutorial/testdata/project/PROJECT

```diff
@@ -1,27 +1,16 @@
 componentConfig: true
 domain: tutorial.kubebuilder.io
-layout: go.kubebuilder.io/v3
-multigroup: true
+layout:
+- go.kubebuilder.io/v3
 projectName: project
 repo: tutorial.kubebuilder.io/project
 resources:
 - api:
     crdVersion: v1
-  group: batch
-  kind: CronJob
-  version: v1
-  webhooks:
-    webhookVersion: v1
-- api:
-    crdVersion: v1
-  group: batch
-  kind: CronJob
-  version: v2
-  webhooks:
-    webhookVersion: v1
-- api:
-    crdVersion: v1
```

Comment:
Seems that is missing generate the CRDs. 
Note that for the other samples we create a shell at least it is helpful to generate the base and then let us know compare diff with master